ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -33,7 +32,7 @@
 public abstract class KeyStoreConfiguration {
 
     @Value.Parameter
-    public abstract Path path();
+    public abstract URI path();
```

Comment:
how does this play with jackson serialization?

- in case path is "foo/bar"
- in case path is "file://foo/bar"
- in case path is "bogusScheme://foo/bar"